docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -59,8 +59,66 @@ alluxio.master.mount.table.root.ufs=<STORAGE_URI>
 - The second property `alluxio.master.mount.table.root.ufs` sets to the URI of the under store to
   mount to the Alluxio root.
   This shared storage system must be accessible by the master node and all worker nodes.
-  Examples include `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`, or 
-  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`.
+  
+  For example, when [HDFS]({{ '/en/ufs/HDFS.html#basic-setup' | relativize_url }})
+  is used as the under storage system, the value of this property can be set to
+  `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`
+  
+  When [Amazon S3]({{ '/en/ufs/S3.html#basic-setup' | relativize_url }})
+  is used as the under storage system, the value can be set to
+  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`
+
+Append thehost name of each node into conf/masters and conf/workers accordingly.
+{% accordion nodeconfig % }
+  {% collapsible Example: master node from Amazon S3 % }
+Append the hostname of each Alluxio master node to a new line into `conf/masters`. 
+Comment out `localhost` if necessary.
+```
+#
```

Comment:
i don't think adding the license header is particularly useful. note that the above steps explaining the properties to be added into alluxio-site.properties did not include the license header.

also because the masters and workers files follow the same format, i don't think it's worth repeating practically identical content.

from these 2 suggested changes, there's not much text to show so using accordions to hide does not seem as useful